zebra/explorer.md

```diff
@@ -13,23 +13,23 @@ https://zcashblockexplorer.com/
+| getblockhashes              | NO.                | As it is used in different places, with different arguments, it need all the fields and options.<br>(noOrphans is always true.)                                                                                                                    | [Zcash](https://zcash.github.io/rpc/getblockhashes.html)                                                                                                                     | [1](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer/blocks/block_warmer.ex#L19) - [2](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer/transactions/transaction_cache.ex#L19) - [3](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer_web/controllers/block_controller.ex#L54) - [4](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer_web/controllers/block_controller.ex#L89)                                                                                                                              |
+| getblock                    | YES, partially.    | At least the following fields are needed as part of `verbose=2` mode:<br>`height`<br>`size`<br>`hash`<br>`time`<br>`tx.vout`<br>We'll need to add `time`, `size`, and `tx.vout` to the existing database indexes, and `size` to the non-finalized state indexes.<br>                                                                                                  | [Zcash](https://zcash.github.io/rpc/getblock.html) - [Zebra](https://doc.zebra.zfnd.org/zebra_rpc/methods/trait.Rpc.html#tymethod.get_block)                                 | [1](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer/blocks/block_warmer.ex#L24) [calls](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer_web/views/block_view.ex#L72) - [2](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer/transactions/transaction_cache.ex#L24) - [3](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer_web/controllers/block_controller.ex#L5) - [4](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer_web/controllers/block_controller.ex#L12) - [5](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer_web/controllers/search_controller.ex#L14) |
```

Comment:
Changed to the RPC fields that are used by the block explorer, it calculates some of the final info itself. Added verbose mode and database info.